src/sentry/south_migrations/0480_removed_scheduled_job.py

```diff
@@ -0,0 +1,1385 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
```

Comment:
@mattrobenolt I assume the best way to do this is to actually let this deploy, and then have #ops remove the table afterwards?

It's currently completely empty, so no data loss concerns.

Reply:
Using is_dangerous gets around the deploy timing issue, another way would be to deploy the migration separately after the model removal goes out.

Reply:
Yeah, maybe I'll just split them up